cluster/cluster_reader.yaml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
This has to be under "test" not under cluster.

Reply:
due to the reasons mentioned at https://github.com/nmstate/kubernetes-nmstate/pull/1052#discussion_r859554382
i think it is more suitable as part of cluster, as it prefer the cluster (independent of cluster-sync and even should be before that)